src/main/java/stirling/software/SPDF/controller/api/UserController.java

```diff
@@ -44,6 +40,14 @@ public class UserController {
     @PreAuthorize("!hasAuthority('ROLE_DEMO_USER')")
     @PostMapping("/register")
     public String register(@ModelAttribute UsernameAndPass requestModel, Model model) {
+
+        if (!userService.isUsernameValid(requestModel.getUsername())) {
+            model.addAttribute(
+                    "invalidUsername",
+                    "Invalid username, Username must only contain alphabet characters and numbers.");
```

Comment:
We can not use this route for error messages as it doesnt enable them to be translated instead something like the messageType in other entries should be used
Since i dont think register actually is used i guess this is fine though, i wonder if we can remove this whole method.. I will look into this

Reply:
Are we going to remove this Register method all together?

Reply:
Maybe but you don't need to address this in your change

Reply:
I wouldn't remove it, maybe sooner or later someone will still need it.